tensorflow/lite/toco/tflite/op_version.cc

```diff
@@ -145,6 +145,7 @@ std::string GetMinimumRuntimeVersionForModel(const Model& model) {
           {{OperatorType::kResizeNearestNeighbor, 1}, "1.13.1"},
           {{OperatorType::kResizeNearestNeighbor, 2}, "1.14.0"},
           {{OperatorType::kResizeNearestNeighbor, 3}, kPendingReleaseOpVersion},
+          {{OperatorType::kResizeNearestNeighbor, 4}, kPendingReleaseOpVersion},
```

Comment:
Note that the version 3 of the `kResizeNearestNeighbor` operator still has a `kPendingReleaseOpVersion` while it is set to "2.3.0" in `tensorflow/lite/tools/versioning/runtime_version.cc`. I left it as it is for now but it may warrant a correction.